index.adoc

```diff
@@ -46,6 +46,13 @@
 
 |===
 
+|===
+|Prospero
+
+|link:prospero[1.1.12.Final]
```

Comment:
One more minor change I missed. I think this should just be `Prospero` instead of the version. This is what we're doing for other components.